sdk/identity/azure-identity/azure/identity/_internal/shared_token_cache.py

```diff
@@ -29,6 +29,7 @@
     import msal_extensions
     from azure.core.credentials import AccessToken
     from .._internal import AadClientBase
+    from azure.identity import AuthenticationRecord
```

Comment:
Why not from ..__auth_record?

Reply:
It's a public class, and I don't want to depend on its internal location.